src/Workspaces/Core/Portable/Workspace/Solution/SolutionInfo.cs

```diff
@@ -138,15 +163,17 @@ public void WriteTo(ObjectWriter writer)
                 // info.Version.WriteTo(writer);
 
                 writer.WriteString(FilePath);
+                writer.WriteGuid(TelemetryId);
```

Comment:
Do we have some serialization version that we have to bump if we change this?

Reply:
IDK what is the versioning guarantees of the serializer API. Is it public?